client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -174,6 +174,37 @@ interface ExternalProps {
 
 type Props = ExternalProps & LocalizeProps;
 
+export const HardBlockingNotice = ( {
+	holds,
+	translate,
+	blockingMessages,
+}: {
+	holds: string[];
+	translate: LocalizeProps[ 'translate' ];
+	blockingMessages: ReturnType< typeof getBlockingMessages >;
+} ) => {
+	const blockingHold = holds.find( ( h ): h is keyof ReturnType< typeof getBlockingMessages > =>
+		isHardBlockingHoldType( h, blockingMessages )
+	);
+	if ( ! blockingHold ) {
```

Comment:
Why here are we checking for false, and in the previous code we were checking for true?

Reply:
Because I cannot necessarily rely on that, this block may be used elsewhere that may not check. I tend to err more on the side of [defensive programming](https://en.wikipedia.org/wiki/Defensive_programming).

Also, Typescript complained ;)